### ISSUE TITLE
Feature: Heroic Library Should Auto Reload After Purchasing Game/DLC

### ISSUE THREAD

**Issue by Nicholas705M** [Heroic-Games-Launcher/HeroicGamesLauncher#2925]
### Problem description

Heroic library does not auto reload after buying game/dlc. The game bought does not show up until after manual reload.

### Feature description

The Heroic library should reload after purchasing any game/dlc to make the game show up in the Library.

**Comment by imLinguin** [comment]
GOG and Amazon launches do not have this feature themselves, not sure about Epic but this might be impossible to do

**Comment by Nicholas705M** [comment]
Understood. Maybe adding an option to check for library updates every 1 min?

**Comment by Gustash** [comment]
@imLinguin I think what we can do is auto refresh all libraries whenever the Library tab is focused

**Comment by Nicholas705M** [comment]
@Gustash I agree with that solution. It'll be much simpler than adding a scheduled update and adding a new setting. I don't know how much data the refresh uses though. I'd assume not too much, so it should be fine

**Comment by flavioislima** [comment]
> @imLinguin I think what we can do is auto refresh all libraries whenever the Library tab is focused

we used to do that but it is a big problem to people with big libraries, especially if you are logged in on all stores and have a couple thousand games. So we removed this feature.

**Comment by Nicholas705M** [comment]
@flavioislima Makes sense. If there is a way to optimize checking for library changes, we might be able to add that back in

**Comment by imLinguin** [comment]
I just found an event sent by GOG's `notifications-pusher` endpoint which should allow this to exist for GOG at least.  
If we would find similar thing for Epic, we could consider the implementation.

In case of Amazon I think this is not needed.
